<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Rawat-PC" failures="1" tests="1" name="CrossBrowserTests3" time="29.671" errors="0" timestamp="2024-06-24T22:51:09 IST">
  <testcase classname="com.opencart.qa.testcases.TestCase3_Login_User_with_incorrect_email_and_password" name="loginUser_with_incorrect_email_and_password" time="0.0">
    <failure type="org.testng.TestNGException" message="
Method public void com.opencart.qa.testcases.TestCase3_Login_User_with_incorrect_email_and_password.loginUser_with_incorrect_email_and_password(java.lang.String,java.lang.String,java.lang.String) requires a @DataProvider named : logindata in class org.testng.annotations.DataProvider">
      <![CDATA[org.testng.TestNGException: 
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- loginUser_with_incorrect_email_and_password -->
</testsuite> <!-- CrossBrowserTests3 -->
